wp-content/themes/ngisweden/functions/shortcodes/deployed_tools_versions.php

```diff
@@ -50,4 +53,5 @@ function ngisweden_deployed_tools_versions_shortcode($atts_raw){
     return $output;
 
 }
-add_shortcode('ngisweden_deployed_tools_versions', 'ngisweden_deployed_tools_versions_shortcode');
+add_shortcode('ngisweden_deployed_tools_versions_sthlm', 'ngisweden_deployed_tools_versions_shortcode');
+add_shortcode('ngisweden_deployed_tools_versions_upps', 'ngisweden_deployed_tools_versions_shortcode');
```

Comment:
See other comment above

```suggestion
add_shortcode('ngisweden_deployed_tools_versions', 'ngisweden_deployed_tools_versions_shortcode');
```

Reply:
With both shortcodes I get:

`This page should only be used as a guide of what to expect. Please use the reported software versions in your data delivery as the definite record of what was used for your analysis.
[ngisweden_deployed_tools_versions]`

But with just the one I get:

`No version information found.`

Reply:
The first one is just because the shortcode doesn't exist, so it's printed as raw text.

The second one is the message from the code above saying that no tool names were parsed. So likely the file wasn't found.

Reply:
So to clarify - the first one doesn't work because here @aanil has removed the original shortcode name `[ngisweden_deployed_tools_versions]` in the code.

> But with just the one I get:

Not entirely clear what you wrote in the content, but maybe you didn't pass the filename? As it stands in the code, even when using the new dual shortcode names you still also need to pass the filename as an argument:

```
[ngisweden_deployed_tools_versions deployed_file_name='deployed_tools.sthlm.version']
```